docker/travis/docker_push.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker tag dessalines/lemmy:travis \
+  dessalines/lemmy:v0.7.28
+docker push dessalines/lemmy:v0.7.28
```

Comment:
These two get overwritten in `deploy.sh` to the correct version.

Reply:
Why not use an env var?

Reply:
You could do either way, but all travis env vars have to get set in the `.travis.yml`, so its either changing that file or this one.